transcom/mymove
# INT-B-18757 HL Begin PPM Upload

## [B-18757](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18757)

## Summary

Uses the existing `updateMTOShipmentPPMDetails.jsx` file, and changes the title if the shipment contains all of the keys in 'changedValues' that match with the object that occurs when a customer fills out the first form of the "Upload PPM Documents" process. Also changed the representation of "has_received_advance: false" from "-" to "No" in the HL.

## How to Test

1. Create a move with a PPM shipment, then sign and send to SC.
2. Approve as SC then go back to customer.
3. Click "Upload PPM Documents". Fill out only the first page.
4. Go back to SC tab and click on Move History Tab, near the top of the page you should see an entry similar to this:
![Screenshot 2024-03-27 at 3 02 02 PM](https://github.com/transcom/mymove/assets/146959895/34412733-7dd3-47ad-856e-68b5eae59158)

Note: If you chose "No" for the "Have you received an advance?" question, you should instead see "No", and no field for advance amount received.
